service/bin/run-in-docker.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env sh
+
+print_help() {
+  echo "Script to run docker containers for Spring Boot Template API service
+
+  Usage:
+
+  ./run-in-docker.sh [OPTIONS]
+
+  Options:
+    --clean, -c                   Clean and install current state of source code
+    --install, -i                 Install current state of source code
+    --param PARAM=, -p PARAM=     Parse script parameter
+    --help, -h                    Print this help block
+
+  Available parameters:
+
+  "
+}
+
+# script execution flags
+GRADLE_CLEAN=false
+GRADLE_INSTALL=false
+
+# TODO custom environment variables application requires.
```

Comment:
unsure about the commented out code in this file and a few more below. I would check through and see what makes sense to keep (if any). TODOs I would have thought can be removed